wait_for_login.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+# Must be exported by the caller:
+# OMERO_USER OMERO_PASS PREFIX
+
+OMERO=/opt/omero/server/OMERO.server/bin/omero
+
+# Wait up to 2 mins
+i=0
+while ! docker-compose exec -T omeroserver $OMERO login -C -s localhost -u "$OMERO_USER" -q -w "$OMERO_PASS"; do
```

Comment:
Wow this needs another hope to live in...

Reply:
Added an issue https://github.com/ome/omero-py/issues/273